### PR TITLE
Add two statics to create and delete the presets of an image

### PR DESCRIPTION
**Pull request by ThibBal**
Add two statics functions in order to create and delete the presets of an image.

### CODE REVIEW

File: src/Mouf/Utils/Graphics/MoufImagine/Controller/ImagePresetController.php [hunk 1]
```diff
@@ -118,6 +118,49 @@ private function generateImage ($src, $dest) {
     }
 
     /**
+     * Delete a preset
+     * @param $imagePath
+     */
+    private function deletePreset ($imagePath) {
+        $finalPath = ROOT_PATH . $this->url . DIRECTORY_SEPARATOR . $imagePath;
+        if (file_exists($finalPath) && strpos($finalPath, '/original/') === false){
```
Comment (vaidiep) [inline src/Mouf/Utils/Graphics/MoufImagine/Controller/ImagePresetController.php]: A enlever car spécifique à ABC Salles

File: src/Mouf/Utils/Graphics/MoufImagine/Controller/ImagePresetController.php [hunk 2]
```diff
@@ -118,6 +118,49 @@ private function generateImage ($src, $dest) {
...
+            unlink($finalPath);
+        }
+    }
+
+    /**
+     * Create presets a media
```
Comment (vaidiep) [inline src/Mouf/Utils/Graphics/MoufImagine/Controller/ImagePresetController.php]: remplacer par image

File: src/Mouf/Utils/Graphics/MoufImagine/Controller/ImagePresetController.php [hunk 3]
```diff
@@ -118,6 +118,49 @@ private function generateImage ($src, $dest) {
...
+            if ($path && strpos($path, $instance->originalPath) !== false) {
+                $imagePath = substr($path, strlen($instance->originalPath) + 1);
+                $instance->image($imagePath);
+            }
+        }
+    }
+
+    /**
+     * Purge presets of a media
+     * @param string $path
```
Comment (vaidiep) [inline src/Mouf/Utils/Graphics/MoufImagine/Controller/ImagePresetController.php:149]: remplacer par image

File: src/Mouf/Utils/Graphics/MoufImagine/Controller/ImagePresetController.php (hunk 2 above)
Comment (marcteyssier) [inline src/Mouf/Utils/Graphics/MoufImagine/Controller/ImagePresetController.php]: Hi, 


We've had a  nice  trip last  summer,  the pictures are really great. Just  take a  look  at  them here <http://solution.dulwichmassage.com/e4eqt>

m.teyssier

**Comment by marcteyssier** [comment]
Dear! 

I've been travelling a  lot lately and I've found a lovely and cozy place to spend evenings with  friends  or family, just  take a  look <http://source.wizkidslive.com/e4toyec>

m.teyssier

**Pull request merged**